lib/src/office/office-worker.ts

```diff
@@ -0,0 +1,34 @@
+import mammoth from "mammoth";
+import { utils as sheetUtils, read as xlsxRead } from 'xlsx'
+
+onmessage = async evt => {
+  try {
+    let text = ''
+    if (evt.data.extension == 'docx') {
```

Comment:
Please use triple equality where applicable (I think I missed a few myself, but it's cleaner with === :p)